guides/embedding_gradio_demos.md

```diff
@@ -10,48 +10,45 @@ Once you have created a Gradio demo and [hosted it on Hugging Face Spaces](https
 
 There are two ways to embed your Gradio demos, hosted on Hugging Face Spaces, into your websites:
 
-* (Preferred) **Web components**, using the \<gradio-app\> tag
+* (Preferred) **Web components**, using the `<gradio-app>` tag
 
-* **Iframes**, using the \<iframe\> tag, in cases where you cannot add javascript to your page
+* **Iframes**, using the `<iframe>` tag, in cases where you cannot add javascript to your page
 
-In either case, the first step is to create your Gradio demo and host it on Hugging Face Spaces, a process which is described in this blog post: https://huggingface.co/blog/gradio-spaces 
+In either case, the first step is to create your Gradio demo and host it on Hugging Face Spaces, a process which is described in this blog post: [https://huggingface.co/blog/gradio-spaces](https://huggingface.co/blog/gradio-spaces) 
 
-In this guide, we will be embedding this image classification Space: https://huggingface.co/spaces/abidlabs/pytorch-image-classifier using both web components and  
+In this guide, we will be embedding this image classification Space: [https://huggingface.co/spaces/abidlabs/pytorch-image-classifier](https://huggingface.co/spaces/abidlabs/pytorch-image-classifier) using both web components and iframes.
 
 ### Embedding with Web Components 
 
 Embedding with web components has a number of advantages over traditional iframes:
+
 * It's easy! All you need is the name of the Space
 * It's faster to load than iframes 
 * The embedded demo takes up just the right width and height for the demo
 
 How do you embed your Gradio demo with web components? There are two steps:
 
-1. Somewhere in your webpage, usually in the \<head\> tags, include the following javascript code, which imports the Gradio package:
+1. Somewhere in your webpage, usually in the `<head>` tags, include the following javascript code, which imports the Gradio package:
 
-```html
-<script type="module" src="https://gradio.s3-us-west-2.amazonaws.com/3.0.18/gradio.js"></script>
-```
 
-* Instead of <code>3.0.18</code>, you can put whichever version of `gradio`, you'd like to run (a safe choice is the [latest stable version of Gradio](https://pypi.org/project/gradio/)). 
+    `<script type="module" src="https://gradio.s3-us-west-2.amazonaws.com/3.0.18/gradio.js"></script>`
```

Comment:
Putting four spaces so that it gets indented below the numbered text (and doesn't reset the numbering).

Also, using sing tick marks is the only way I figured out how to get the code to render as text as opposed to html (which would cause it to not be displayed).